src/Packetery/Module/Checkout.php

```diff
@@ -1075,7 +1075,7 @@ public function isPickupPointCarrier( string $carrierId ): bool {
 	private function getPostDataIncludingStoredData( string $chosenShippingMethod ): array {
 		$checkoutData      = $this->httpRequest->getPost();
 		$savedCheckoutData = get_transient( $this->getTransientNamePacketaCheckoutData() );
-		if ( ! is_array( $savedCheckoutData[ $chosenShippingMethod ] ) ) {
+		if ( empty( $savedCheckoutData ) || ! is_array( $savedCheckoutData[ $chosenShippingMethod ] ?? false ) ) {
```

Comment:
Zdá se, že by stačilo samotné `?? false`, ale není to přehledné. Co to zapsat jako `if ( ! is_array( $savedCheckoutData ) || ! is_array( $savedCheckoutData[ $chosenShippingMethod ] ) )` ?